src/ILVerify/tests/TestDataLoader.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Internal.IL;
+using Internal.TypeSystem.Ecma;
+
+namespace ILVerify.Tests
+{
+    /// <summary>
+    /// Helper method to load the binaries generated based on the il code, which drive the tests
+    /// </summary>
+    static class TestDataLoader
+    {
+        /// <summary>
+        /// The folder which contains the core assembly (containing System.Object, etc.)
+        /// </summary>
+        static string COREASSEMBLYPATH = @"..\..\..\..\..\..\bin\Product\Windows_NT.x64.Debug\CoreTestAssembly\\";
+
+        /// <summary>
+        /// The folter with the binaries which are compiled from the test driver IL Code
+        /// </summary>
+        static string TESTASSEMBLYPATH = @"..\..\..\ILTests\";
```

Comment:
.ilproj is not supported by the .NET SDK out of the box, so there is no mainstream documentation for it. .NET SDK includes targets based on the project extension. For example, you can find code like this in Sdk.targets from the .NET SDK:

```
    <LanguageTargets Condition="'$(MSBuildProjectExtension)' == '.csproj'">$(MSBuildToolsPath)\Microsoft.CSharp.targets</LanguageTargets>
    <LanguageTargets Condition="'$(MSBuildProjectExtension)' == '.vbproj'">$(MSBuildToolsPath)\Microsoft.VisualBasic.targets</LanguageTargets>
```

Notice that there is no condition for .ilproj. You have to take of the msbuild targets for .ilproj yourself. I assume that it is what the VS is complaining about.

The .ilproj files work in the CoreRT build today because of our own `buildtools` extension that setups the targets for .ilproj: https://github.com/dotnet/buildtools/blob/03988127f489bcd1cf58d71131ac628620d53de9/src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets#L106

I doubt that you would be able to include the buildtools ilproj targets into your project because of your have started using latest .NET CLI.

I think it should be ok to depend on manual building for now, and add the project file once #3682 is fixed.